resources/js/profile-page/daily-challenge.tsx

```diff
@@ -122,26 +123,30 @@ export default class DailyChallenge extends React.Component<Props> {
       return null;
     }
 
+    const playedToday = this.props.stats.last_update != null && moment.utc(this.props.stats.last_update).isSame(Date.now(), 'day');
```

Comment:
I *think* this is gonna do timezones correctly:

https://momentjs.com/docs/#/query/is-same/

> If the two moments have different timezones, the timezone of the first moment will be used for the comparison.

and it looked correct when I messed around with my timezone...

Reply:
yeah that seems correct (until someone complains it's not)

alternatively `new Date(...).toISOString()` spits out iso8601 string which has utc timezone and can then be sliced _appropriately_ (not sure if better)

Reply:
> and can then be sliced _appropriately_ (not sure if better)

I uh... am not confident in my ability to do so, so I'd rather not if that's fine with you :sweat_smile: